### PR TITLE
fix(orchestration,link-modules,pricing,types): fix shippingprofile errror outside of core + change link alias name

### PR DESCRIPTION
**Pull request by riqwan**

### CODE REVIEW

File: packages/link-modules/src/definitions/product-variant-price-set.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const ProductVariantPriceSet: ModuleJoinerConfig = {
         serviceName: LINKS.ProductVariantPriceSet,
         primaryKey: "variant_id",
         foreignKey: "id",
-        alias: "prices",
+        alias: "price",
         isList: false,
```
Comment (adrien2p) [inline packages/link-modules/src/definitions/product-variant-price-set.ts]: comment: I think it is not a list by default right?

**Review by carlos-r-l-rodrigues — APPROVED** [review]
LGTM

File: packages/pricing/src/services/pricing-module.ts [hunk 2]
```diff
@@ -62,7 +62,7 @@ export default class PricingModuleService<
   @InjectManager("baseRepository_")
   async calculatePrices(
     priceSetIds: string[],
-    pricingContext: PricingContext,
+    pricingContext: PricingContext = {},
```
Comment (adrien2p) [inline packages/pricing/src/services/pricing-module.ts]: comment l112 should we serialize instead?

**Review by adrien2p — APPROVED** [review]
LGTM

File: packages/link-modules/src/definitions/product-variant-price-set.ts (hunk 1 above)
Comment (carlos-r-l-rodrigues) [reply packages/link-modules/src/definitions/product-variant-price-set.ts]: yeah, that could be removed.

File: packages/pricing/src/services/pricing-module.ts (hunk 2 above)
Comment (riqwan) [reply packages/pricing/src/services/pricing-module.ts]: doesn't seem to work with non mikroorm entities

File: packages/pricing/src/joiner-config.ts [hunk 3]
```diff
@@ -32,6 +32,18 @@ export const joinerConfig: ModuleJoinerConfig = {
...
+      args: {
+        methodOverride: "calculatePrices",
+      },
+    },
+    {
+      name: "calculated_price",
+      args: {
+        methodOverride: "calculatePrice",
+      },
+    },
```
Comment (olivermrbl) [inline packages/pricing/src/joiner-config.ts]: thought:

I think for the first iteration and to not rush into a pattern, the call to calculate prices should be done separately from the Remote Query call. 

So right now, we will only use the Remote Query engine as a way to collect raw data for further processing.

Comment (riqwan) [reply packages/pricing/src/joiner-config.ts]: +1. This won't be committed. I'm just jesting with the idea and using this PR to create snapshots.

**Pull request merged**